src/ucp/api/ucp.h

```diff
@@ -1552,6 +1555,32 @@ ucs_status_t ucp_ep_create(ucp_worker_h worker, const ucp_ep_params_t *params,
                            ucp_ep_h *ep_p);
 
 
+/**
+ * @ingroup UCP_ENDPOINT
+ * @brief Modify endpoint parameters.
+ *
+ * This routine modifies @ref ucp_ep_h "endpoint" created by @ref ucp_ep_create
+ * or @ref ucp_listener_accept_callback_t.
```

Comment:
Need to say that only some of the parameters can be modified and that the documentation of ucp_ep_params_t  species which exactly

Reply:
added a `@note` below

Reply:
Describe the use case for the function